main.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(prog = 'main.py',
+                                     description = 'parsing input/output')
```

Comment:
`argparse` automatically sets the program name to the command name as seen on the command line, so setting `prog` is not necessary. You should also create a more meaningful description. What will this program do?